style.css

```diff
@@ -0,0 +1,157 @@
+:root {
+  --header-mobile-color: #3a4a42;
+  --header-desktop-color: #344563;
+  --headline-color-1: #172b4d;
+  --headline-color-2: #36b37e;
+  --background-color: #fff;
+}
+
+/* Reset the default browser styles */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+}
+
+/* Global styles */
+body {
+  background: var(--background-color);
+}
+
+li {
+  list-style: none;
+}
+
+/* Navbar */
+nav {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  height: 48px;
+  top: 44px;
+}
+
+/* Hide the nav bar on mobile */
+.navbar ul {
+  display: none;
+}
+
+/* Hamburger menu */
+.hamburger_toggle {
+  display: block;
+  cursor: pointer;
+  margin-right: 1rem;
+  margin-top: 0.75rem;
+  align-items: center;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 2rem;
+  height: 1.75rem;
+  background: var(--background-color);
+  border: none;
+}
+
+.navbar li {
+  margin: 0 1rem;
+}
+
+.navbar li a {
+  color: var(--header-mobile-color);
+}
+
+/* Section headline */
+.headline,
+.headline-title,
+.headline-paragraph,
+.headline-social {
+  margin-left: 27px;
+  margin-right: 27px;
+  text-align: center;
+  justify-content: center;
+}
```

Comment:
- [ ] As mentioned in the previous review, It would be nice to have a class name for these properties and use it for all the elements that these properties need to be applied.
E.g.
```sh
.headlines {
  margin-left: 27px;
  margin-right: 27px;
  text-align: center;
  justify-content: center;
}
```
And add the `headlines` to all elements class attributes in your HTML file that these properties need to be applied.

Reply:
Ooooh good point, all of that are in one section with the class .headline . Thanks @LionRouge1